docs/load-test.md

```diff
@@ -0,0 +1,29 @@
+## Test simulations
+
+The platform was tested using following test simulations:
+
+- Message publishing simulation
+- Create and get client simulation
+
+Purpose of these simulations is to see how responsive platform will be under
+heavy loads.
+
+### Message publishing simulation
```

Comment:
Drop the "simulation", keep the description. Results should follow the description, once available.